sim/field.ts

```diff
@@ -99,15 +99,14 @@ export class Field {
 	}
 
 	effectiveWeather() {
-		if (this.suppressingWeather()) return '';
-		return this.weather;
+		return this.suppressingWeather() ? '' : this.weather;
```

Comment:
No point in changing this....

Reply:
```suggestion
		if (this.suppressingWeather()) return '';
		return this.weather;
```

Well. There's some argument about readability. Shorter is often more readable. 🤔